app/javascript/packages/verify-flow/verify-flow.tsx

```diff
@@ -36,6 +36,8 @@ export interface VerifyFlowValues {
   ssn?: string;
 
   password?: string;
+
+  dob?: string;
```

Comment:
would it make things easier if we serialized as milliseconds from the epoch? then JS code is as easy as `new Date(dob)` Or would the `toLocaleDateString` still need a timezone?